packages/client/src/components/form/FormFieldGenerator.tsx

```diff
@@ -131,10 +131,7 @@ const FormItem = styled.div<{
 }>`
```

Comment:
ignoreBottomMargin when is this prop needed? Suggest we remove

Reply:
It is used, whenever the SUBSECTION component is used.

Reply:
ok cool. This Subsection might not be required when we move addresses into model 
<img width="454" alt="image" src="https://user-images.githubusercontent.com/46478402/204573192-57a6c993-2cc6-4bb6-9254-a9b69ed470b9.png">